CUJExecutor/fileToDump

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+# INPUT: a .prof or .txt file
+# OUTPUT: a method-class dump. 
+# If a .txt file is passed, it is enterpretted as the dump to be returned.
```

Comment:
Typo `enterpretted`?  Instead of interpretted, the file is simply passed through.